.github/workflows/maven_dependency_scan.yml

```diff
@@ -3,11 +3,14 @@ on:
   push:
     branches:
       - develop
+      - release/*
     paths:
       - 'pom.xml'
+  schedule:
+    - cron: '0 2 * * 1' # Weekly on Monday at 02:00 AM UTC
```

Comment:
A schedule is not required. The command 'snyk monitor' used in the dependency scan template will gather dependencies for Snyk where they are already tested regularly.